src/app/components/transactionsRequests/ContractCallRequest.tsx

```diff
@@ -269,23 +269,28 @@ export default function ContractCallRequest(props: ContractCallRequestProps) {
 
   const renderPostConditionsCard = () => {
     const { postConds } = extractFromPayload(request);
+    let coinInfo;
     return postConds?.map((postCondition, i) => {
       switch (postCondition.conditionType) {
         case PostConditionType.STX:
-          return <StxPostConditionCard key={i} postCondition={postCondition} />;
+          return <StxPostConditionCard key={postCondition.type} postCondition={postCondition} />;
```

Comment:
i'm not 100% sure on what postCondition.type is, but the react key should be an identifier, unique within the list. if it's possible or likely for the array to have more than one item with the same postCondition.type, then it's not a good choice for the key